src/Umbraco.Web.UI.Client/src/packages/core/entity-action/common/delete/delete.action.ts

```diff
@@ -21,12 +25,20 @@ export class UmbDeleteEntityAction extends UmbEntityActionBase<MetaEntityActionD
 		const item = data?.[0];
 		if (!item) throw new Error('Item not found.');
 
+		const headline = this.args.meta.confirm?.headline ?? '#actions_delete';
+		const content = this.args.meta.confirm?.content ?? '#defaultdialogs_confirmdelete';
+		const keysFromString = this.#localize.getKeysFromString(content);
```

Comment:
I'm not sure i get what is going on here? :-O

Reply:
I will add some comments :D I hope it will help.